css/style.css

```diff
@@ -1 +1,1091 @@
+  margin-top: 8px;
+  margin-bottom: 40px;
+}
+
+.footer-btn-box {
+  margin-bottom: 40px;
+}
+
+.btn-footer {
+  background-color: rgba(88, 101, 242, 1);
+}
+
+/* social-networks Footer */
+
+.network-item {
+  margin: 0 auto;
+  background-color: rgba(194, 195, 197, 0.15);
+  width: 50px;
+  height: 50px;
+  margin-bottom: 20px;
+}
+
+.network-item img {
+  display: block;
+  margin: 0 auto;
+  padding: 14px 10px;
+  width: 26px;
+}
+
+/* footer-nav section */
+
+.footer-border {
+  border-top: 1px solid rgba(67, 67, 67, 1);
+  padding: 19px 0;
+}
+
+.footer-logo {
+  font-size: 22px;
+  line-height: 30px;
+  gap: 16px;
+}
+
+.footer-logo img {
+  width: 24px;
+  height: 26px;
+}
+
+.footer-logo-box p {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  font-size: 14px;
+  line-height: 22px;
+  text-align: right;
+}
+
+/* Footer Pages */
+
+.nav-pages-item a {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  font-size: 14px;
+  line-height: 22px;
+  text-align: right;
+}
+
+/************************* Catalog Page **************************/
+
+/* Breadcrumbs */
+
+.breadcrumbs {
+  background-color: var(--secondary-color);
+  padding-top: 77px;
+  padding-bottom: 77px;
+  text-align: center;
+}
+
+.breadcrumbs * {
+  background-color: transparent;
+}
+
+.breadcrumbs ul {
+  margin-top: 12px;
+  display: flex;
+  justify-content: center;
+}
+
+.breadcrumbs a {
+  padding: 0 8px;
+}
+
+.breadcrumbs a,
+.breadcrumbs span {
+  font-size: 14px;
+  line-height: 20px;
+  justify-content: center;
+}
+
+.inactive-link a {
+  color: var(--inactive-color);
+  border-right: 2px solid var(--inactive-color);
+}
+
+/* NFT Collection */
+
+.nft-collection {
+  background-color: var(--main-background-color);
+  padding-top: 100px;
+  padding-bottom: 200px;
+}
+
+.nft-collection h2 {
+  text-align: left;
+  margin-bottom: 40px;
+  background-color: transparent;
+}
+
+/* Call Action section - coloful */
+
+.CTA-section {
+  background-color: var(--secondary-color);
+}
+
+.call-action-catalog {
+  background-color: var(--secondary-color);
+  position: relative;
+  top: -100px;
+  margin: 0 255px;
+  padding: 40px 0 48px;
+  border-radius: 8px;
+  background: linear-gradient(
+    to left,
+    rgba(35, 51, 222, 1),
+    rgba(10, 190, 102, 1)
+  );
+}
+
+.catalog-cta-title {
+  background-color: transparent;
+}
+
+.catalog-cta-text {
+  margin-top: 4px;
+  margin-bottom: 22px;
+  text-align: center;
+  font-family: var(--text-font);
+  color: rgba(247, 247, 249, 1);
+  background-color: transparent;
+}
+
+.cta-catbtn-box {
+  text-align: center;
+  background-color: transparent;
+}
+
+.cta-btn-link {
+  padding: 12px 24px;
+  background-color: rgba(88, 101, 242, 1);
+  text-align: center;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.cta-btn-link img {
+  background-color: transparent;
+  width: 24px;
+}
+
+/* Footer */
+
+.catalog-footer {
+  background-color: var(--secondary-color);
+  padding-top: -40px;
```

Comment:
Padding cannot be negative:

In CSS, padding represents the space between the content and the border of an element and cannot be negative. Negative padding would not make sense, as it would imply that the content overlaps the border, which is not possible.

Reply:
#fixed